server/app/views/BaseHtmlView.java

```diff
@@ -77,7 +77,7 @@ protected static ButtonTag asRedirectButton(ButtonTag buttonEl, String redirectU
   protected static ButtonTag makeSvgTextButton(String buttonText, Icons icon) {
     return TagCreator.button()
         .with(
-            Icons.svg(icon, 18).withClasses(Styles.ML_1, Styles.INLINE_BLOCK, Styles.FLEX_SHRINK_0),
+            Icons.svg(icon, 20).withClasses(Styles.ML_1, Styles.INLINE_BLOCK, Styles.FLEX_SHRINK_0),
```

Comment:
The intention was that SVG buttons would render their icons as 18px (from mocks). Is it possible to add the appropriate styling here such that it will render as 18px?

Reply:
This is one is tricky. We are heavily using tailwindcss and it doesn't have size for 18px, only 16px and 20px: https://tailwindcss.com/docs/width#fixed-widths Do you know if have examples of setting size explicitly on elements, without using tailwind?

Reply:
Once we're on Tailwind 3, we can set the value explicitly like this: https://tailwindcss.com/docs/width#arbitrary-values. That said, we're not using Tailwind 3 yet.

The Tailwind docs semi-imply that we could add another size if needed, but that might not be worth the time to add. I'd be in favor of setting an inline style for this one (e.g. `style="width:18px; height:18px"`)

Reply:
Sg, set width/height using styles.